dev_tools/profiling/benchmark_simulators_test.py

```diff
@@ -13,16 +13,18 @@
 # limitations under the License.
 
 """Tests for the simulator benchmarker."""
-
+import cirq
 from dev_tools.profiling import benchmark_simulators
 
 
+@cirq.testing.skip_if_module_not_exists(module="cirq.google")
```

Comment:
A little uneasy about skipping these, but it's dev_tools so I'm not vehemently opposed.